rclpy_components/setup.py

```diff
@@ -0,0 +1,27 @@
+from setuptools import setup
+
+package_name = 'rclpy_components'
+
+setup(
+    name=package_name,
+    version='0.0.0',
+    packages=[package_name],
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    maintainer='root',
+    maintainer_email='root@todo.todo',
```

Comment:
Let's populate this. Feel free to add my info.
Also, add your information as an author!
```suggestion
    author='YOUR_NAME',
    author_email='OPTIONALLY YOUR EMAIL'
    maintainer='Jacob Perron',
    maintainer_email='jacob@openrobotics.org',
```